x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/datafeed_step/components/query/description.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo, FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiFormRow } from '@elastic/eui';
+
+interface Props {
```

Comment:
Can you try if you don't need this custom prop interface at all? I think I remember `FC` already has `children` as a prop and extends what you pass in.